cartographer/io/internal/mapping_state_serialization.cc

```diff
@@ -197,8 +212,9 @@ void WritePbStream(
     ProtoStreamWriterInterface* const writer) {
   writer->WriteProto(CreateHeader());
   writer->WriteProto(SerializePoseGraph(pose_graph));
-  writer->WriteProto(
-      SerializeAllTrajectoryBuilderOptions(trajectory_builder_options));
+  writer->WriteProto(SerializeAllTrajectoryBuilderOptions(
```

Comment:
Perhaps change remove the "all" from the function name because it's no longer the case?

Reply:
good catch :-)